packages/lab/src/calendar/Calendar.tsx

```diff
@@ -1,46 +1,72 @@
 import { makePrefixer } from "@salt-ds/core";
 import { clsx } from "clsx";
-import { forwardRef, useCallback } from "react";
+import {
+  type ComponentPropsWithoutRef,
+  type ReactNode,
+  forwardRef,
+  useCallback,
+} from "react";
 import {
   CalendarCarousel,
   type CalendarCarouselProps,
 } from "./internal/CalendarCarousel";
 import { CalendarContext } from "./internal/CalendarContext";
-import {
-  CalendarNavigation,
-  type CalendarNavigationProps,
-} from "./internal/CalendarNavigation";
 import { CalendarWeekHeader } from "./internal/CalendarWeekHeader";
-import { useCalendar, type useCalendarProps } from "./useCalendar";
+import {
+  type UseCalendarMultiSelectProps,
+  type UseCalendarOffsetProps,
+  type UseCalendarRangeProps,
+  type UseCalendarSingleProps,
+  useCalendar,
+} from "./useCalendar";
 
 import { useComponentCssInjection } from "@salt-ds/styles";
 import { useWindow } from "@salt-ds/window";
 
-import { DateFormatter, getLocalTimeZone } from "@internationalized/date";
+import { DateFormatter } from "@internationalized/date";
 import calendarCss from "./Calendar.css";
-import { getCurrentLocale } from "./internal/utils";
 
-export type CalendarProps = useCalendarProps & {
+export interface CalendarBaseProps extends ComponentPropsWithoutRef<"div"> {
+  children?: ReactNode;
```

Comment:
I only realize this is inserted before the content when I dig into the code, and `<CalendarNavigation/>` is being set in the story. Not 100% certain whether there's some alternative way to not make this a little confusing, e.g. `heading` & `foot`, `position` (north/west/east/south/center) prop from `BorderItem`

Reply:
The reason it was split out, was so we could re-use Calendars together and they don't all have their own navigations.

CalendarNavigation is a component that can be composed within the DatePicker children, it appears in DatePickerRangePanel and DatePickerSinglePanel, as these are pre-canned patterns which contain the Calendars.
The intention is that folks can compose  DatePickerRangePanel/DatePickerSinglePanel with their own content (like we do in the custom panel example). Alternatively, folks can roll their own.

It's theoretically possible to create a calendar containing 12 calendars (or less) and then a Calendar Navigation with a `step` of 12 that advances the whole view by a calendar year.

Reply:
Yes, i understand the need for composability. It's more about API structure. Everyone would have different idea for `children` of a Calendar ? With the current structure, should it be something like `header`? Or should it not be part of Calendar all together, but be a completely separate piece?

Reply:
We had opinion built into this component before by embedding one navigation to rule them all and we had props to control that component.

This approach removes those props and gives folks choice over the navigation element.

I can 't see any other use for children and don't think we should push any other elements to Calendar.
If folks can highlight a use-case then we could refactor our API around that as any change would just see us deprecate children and use a prop instead. 

However, what we would then use children for ? Perhaps there is a known use-case we have not yet heard about?